less/utilities.less

```diff
@@ -53,3 +53,15 @@
 .affix {
   position: fixed;
 }
+
+
+// Adds ellipsis if text overflows
+// -------------------------
+
+.ellipsis-if-text-overflows {
+  display: inline-block;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  -o-text-overflow: ellipsis;
```

Comment:
For future reference, we use [Autoprefixer](https://github.com/postcss/autoprefixer), so explicit prefixes such as this should normally not be included in our Less/Sass code.